File: fbb.r
fbb <- function() {
    for(num in 1:100) {
        if (num%%105 == 0) {print("FizzBuzzBizz")}
        else if (num%%15 == 0) {print("FizzBuzz")}
        else if (num%%35 == 0) {print("BuzzBizz")}
        else if (num%%21 == 0) {print("FizzBizz")}
        else if (num%%3 == 0) {print("Fizz")}
        else if (num%%5 == 0) {print("Buzz")}
        else if (num%%7 == 0) {print("Bizz")}
        else {print(num)}
    }
}

File: fbb.vb
Module FizzBuzz
    Sub Main()
        Dim I As Integer
        For I = 1 to 100
            If I mod 135 = 0 Then
                Console.WriteLine("FizzBuzzBizz")

            ElseIf I mod 15 = 0 Then
                Console.WriteLine("FizzBuzz")
            ElseIf I mod 35 = 0 Then
                Console.WriteLine("BuzzBizz")
            ElseIf I mod 21 = 0 Then
                Console.WriteLine("FizzBizz")

            ElseIf I mod 3 = 0 Then
                Console.WriteLine("Fizz")
            ElseIf I mod 5 = 0 Then
                Console.WriteLine("Buzz")
            ElseIf I mod 7 = 0 Then
                Console.WriteLine("Bizz")

            Else
                Console.WriteLine(I)
            End If
        Next
    End Sub
End Module
